_slides/09-ipatriots.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello World!
```

Comment:
Looks good, could you mention "Hello Git" as well?

Reply:
Hello Git